cmd/yesiscan/Makefile

```diff
@@ -1,9 +1,13 @@
 .PHONY: all build release
 
+AUTO := $(shell cat ../../.autoConfigURI 2>/dev/null || echo '')
+COOKIE := $(shell cat ../../.autoConfigCookiePath 2>/dev/null || echo '')
```

Comment:
Good idea. I added docs in the latest version.

It's better like this because you can just create those files once and not ever type it again. But you are welcome to override variables with `make` magic if you prefer.